pkg/commands/models/stash_entry.go

```diff
@@ -1,17 +1,49 @@
 package models
 
-import "fmt"
+import (
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/jesseduffield/lazygit/pkg/utils"
+)
 
 // StashEntry : A git stash entry
 type StashEntry struct {
 	Index int
+	Time  *time.Time
```

Comment:
I'd make this consistent with how we do this for branches, and have a `Recency string` field. No accessors at all needed. Clients just work with the field directly, it will be the empty string if the loader failed to determine it for some reason (which can't really happen except when there are bugs in git).

Reply:
I understand the point; I just thought that storing the time itself instead of the recency would have given us more control on the way we show it. For example, it would be simpler to change its format when we decrease the zoom, as we discussed in #3167. I'll change it as you suggested, and we'll touch again on the follow-up PR 👍🏻.

Reply:
You are right that converting the timestamp to a string could also be seen as presentation stuff and shouldn't be in the model. For branches I'm guessing it was done like this for performance reasons; there's no reason to do the conversion again every time we render the view. We can reconsider this, but for now I'd just do it the way it's done for branches so that the two are consistent.